docs/user/exams/instructors_guide.rst

```diff
@@ -173,12 +173,18 @@ During the exam creation and configuration, you can create your exam and configu
     .. note::
        If you do not press this button, the students quiz exercises will not be graded.
 
+- After the :ref:`exam conduction <exam_conduction>` ends, you can click on |assess_unsubmitted_student_exams|. This action will automatically evaluate the text and modelling exercise submissions with 0 points for the unsubmitted student exams. 
```

Comment:
modelling --> modeling
Please also mention that this will also automatically assess empty submissions with 0 points